Gemfile

```diff
@@ -17,6 +17,9 @@ source 'https://rails-assets.org' do
   gem 'rails-assets-datetimepicker'
 end
 
+# Handles text messages and voice calls
+gem 'twilio-ruby'
```

Comment:
Please pin this to the current version, allowing point releases. I have had issues where gems have changed unexpectedly and caused issues.